data/h2_storage_materials/meta.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: h2_storage_reversible_hydrides
+description: synthetic procedures, experimental and theoretical h2 capacities of hydrides
+targets:
+    - id: h_weight_density_theory
+      description: theoretical hydrogen storage capacity
+      units: wt%
+      type: continuous
+      names:
+          - theoretical hydrogen storage weight density
+    - id: h_weight_density_experiment
+      description: experimental hydrogen storage capacity
+      units: wt%
+      type: continuous
+      names:
+          - experimental hydrogen storage capacity
+    - id: synthetic_information
+      description: brief description of synthetic procedure
+      units:
+      type: string
+      names:
+          - synthetic procedure
```

Comment:
This is an interesting column. I felt it is more of an identifier (or something we ad to an identifier); what do you think?